networkx/testing/utils.py

```diff
@@ -1,3 +1,5 @@
+import warnings
+
```

Comment:
Maybe move the import inside the `almost_equal` def so that it is also deleted when we remove `almost_equal` in 3.0.

Reply:
Done.